lang/shared/src/main/scala/com/wavesplatform/lang/v1/compiler/Decompiler.scala

```diff
@@ -181,8 +181,8 @@ object Decompiler {
             val name = extractFunctionName(ctx, header)
             argsCoeval.map(as => out(s"$name(${as.mkString(", ")})", i))
         }
-      case _: Terms.ARR     => ??? // never happens
-      case _: Terms.CaseObj => ??? // never happens
+      case _: Terms.ARR       => ??? // never happens
```

Comment:
How about processing ARR too?

Reply:
Maybe. But there is no ARR in compiled user source code.